script/build.js

```diff
@@ -690,6 +691,40 @@ smith.use(redirect({
   '/education/apply-for-education-benefits/': '/education/apply/'
 }));
 
+/*
+Add nonce attribute with substition string to all inline script tags
+Convert onclick event handles into nonced script tags
+*/
+smith.use((files, metalsmith, done) => {
+  Object.keys(files).forEach((file) => {
+    if (path.extname(file) !== '.html') return;
+
+    const data = files[file];
+    const dom = new jsdom.JSDOM(data.contents.toString());
```

Comment:
I wonder if something like cheerio would fit better here? It doesn't look like you need something as in depth as jsdom. Though I realize we already depend on it, so it's easier to pull in.

https://github.com/cheeriojs/cheerio

Reply:
@jbalboni Is this going to have conflicts with the work being done for Webpack 4?

Reply:
@ncksllvn No, shouldn't have any conflicts with the Webpack changes.